Week1/homework/js-exercises/TypeChecker.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+let book = 'It was the best of times';
+let song = 'In the year 2525';
+let person = {firstName:"John", lastName:"Wick", age:40, eyeColor:"green"};
+let animal = {type:'cat', name:'Silly', color:'white and black'};
+
+console.log(typeof 'book');
+console.log(typeof 'animal');
+
+if (typeof 'book' === typeof 'song') {
```

Comment:
Hmm.. couple things here. Firstly, you could use `const` here instead of `let` since we aren't reassigning these variables. Secondly, I see a mistake in the `if-else-if` blocks which causes errors in your program. Can you figure it out and fix it? 😉